efficientdet/keras/efficientdet_arch_keras.py

```diff
@@ -696,6 +681,62 @@ def call(self, feats):
     return feats
 
 
+class ResampleFeatureAdder(tf.keras.layers.Layer):
```

Comment:
I won't approve such modifies that just wrapper it with a keras layer and change code positions.

Reply:
I created this layer cause I don't know how I can implement it without it. If you know how to create it without keras layer, please share your ideas.

Reply:
Exist code has implemented it.

Reply:
The reason of this PR is the existing code didn't work for me, especially I couldn't inference with existing keras model.
> You should add resample_feature_adder variable scope to tf1.x for unify variables' names if you want to add the layer.
you shouldn't, I fixed it, that why I am able to restore the model from the checkpoint.

Reply:
It may just your problem.
I can't meet any bug when I run test cases.

Reply:
> It may just your problem.

agree

> I can't meet any bug when I run test cases.

sure, but in the current implementation, keras model ran only under the `tf.Session` and with `sess.run(tf.global_variables_initializer())`, but in my case, I would like to run them without session.

Reply:
> > It may just your problem.
> 
> agree
> 
> > I can't meet any bug when I run test cases.
> 
> sure, but in the current implementation, keras model ran only under the `tf.Session` and with `sess.run(tf.global_variables_initializer())`, but in my case, I would like to run them without session.

```
from keras.efficientdet_arch_keras import efficientdet
import tensorflow as tf

model = efficientdet('efficientdet-d0')
ckpt = tf.train.load_checkpoint('checkpoint_path')
for var in model.variables:
  var.assign(ckpt.get_tensor(var.name.split(':')[0]))
model.save_weights('./test.h5', save_format='h5')
model.load_weights('./test.h5')
```
Sure, it doesn't need Session anymore.